CLI/Settings/appsettings.a113296818.json

```diff
@@ -1,7 +1,9 @@
 ﻿{
   "ExporterSettings": {
-    "SharepointUrl": "https://pims-test.zspi.t-systems.com/",
-    "ApiBaseUrl": "https://pims-prov-app-test.zspi.telekom.de/",
+    //"SharepointUrl": "https://pims-test.zspi.t-systems.com/",
+    //"ApiBaseUrl": "https://pims-prov-app-test.zspi.telekom.de/",
```

Comment:
Don't commit production values, because it will be accidentally run on prod. Maybe we need to discuss a more secure option to control live runs.

Reply:
(Actually meant the next two lines.)